Fixed world being disposed multiple times (#952)
spatialos/gdk-for-unity: CHANGELOG.md

## Unreleased

### Fixed

> - Fixed a bug where the `World` would get disposed multiple times, if you stopped the application inside the Editor while the worker still gets created.

Reviewer: ```suggestion
- Fixed a bug where a worker's `World` could get disposed multiple times if you stopped the application inside the Editor while the worker is being created.
```
